AGENTS.md

```
@@ -0,0 +1,117 @@
+## ソリューション構成
+
+| プロジェクト | パス | 役割 |
+|---|---|---|
+| Illustra | `src/Illustra.csproj` | メインアプリケーション |
+| Illustra.Shared | `src/Shared/Illustra.Shared.csproj` | 共有ライブラリ |
+| Illustra.MCPHost | `src/MCPHost/Illustra.MCPHost.csproj` | MCP サーバー機能 |
+| Illustra.Tests | `tests/Illustra.Tests.csproj` | テスト（NUnit + Moq） |
+
+## ソースコード構造（src/ 配下）
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)